gamemodes/terrortown/gamemode/client/cl_help/populate_administration.lua

```diff
@@ -0,0 +1,112 @@
+local materialIcon = Material("vgui/ttt/vskin/helpscreen/administration")
+
+local function PopulateHUDPanel(parent)
+	local form = CreateForm(parent, "header_hud_administration")
+
```

Comment:
maybe someone else can comment this as well. But I added the empty line because these are different types of variables. One is the form, the other is the data needed to create the content.

Reply:
it is okay to stay here i'd say